src/mumble/GlobalShortcut_win.cpp

```diff
@@ -899,3 +903,24 @@ QString GlobalShortcutWin::buttonName(const QVariant &v) {
 bool GlobalShortcutWin::canSuppress() {
 	return bHook;
 }
+
+bool GlobalShortcutWin::areScreenReadersActive() {
+	// This list contains valid executables we consider to be 'screen readers'.
+	// Todo: perhaps make this a configuration option in mumble to let users dynamically add executables.
+	const QStringList executables = {"nvda.exe", "jfw.exe"};
+	const auto snapshot = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0);
+	if (snapshot != INVALID_HANDLE_VALUE) {
+		PROCESSENTRY32 p;
+		p.dwSize = sizeof(p);
+		auto ok = Process32First(hSnap, &processEntry);
+		while (ok) {
+			if (executables.contains(QString::fromWCharArray(p.szExeFile)) {
```

Comment:
```suggestion
			if (executables.contains(QString::fromWCharArray(p.szExeFile))) {
```

Reply:
My fault, I wrote 2 parentheses instead of 3 in https://github.com/mumble-voip/mumble/pull/3896#issuecomment-554086656, sorry.